### DISCUSSION TITLE
Using btop with "-t"

### DISCUSSION THREAD

**Discussion by dieBakterie** [prasanthrangan/hyprdots#1312] [General]
when i change the first entry in sysmonlaunch.sh to btop -t it just skips it and doesn't execute it. only with btop it works fine

**Comment by kRHYME7** [comment]
Use the config instead

```
force_tty = True

```

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/6bc686ad-077d-4a30-894c-015354c2f910)

**Comment by dieBakterie** [comment]
Ok thanks
